CredentialProvider.Microsoft/Logging/PluginConnectionLogger.cs

```diff
@@ -21,9 +23,11 @@ protected override void WriteLog(LogLevel logLevel, string message)
         {
             // intentionally not awaiting here -- don't want to block forward progress just because we tried to log.
             connection.SendRequestAndReceiveResponseAsync<LogRequest, LogResponse>(
-                MessageMethod.Log,
```

Comment:
Does this logger honor the [SetLogLevelRequest](https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.Protocol/Plugins/Messages/SetLogLevelRequest.cs ) message from NuGet?

I've seen some extra logging on NuGet side that just gets swallowed.

Reply:
It should, yes. That's handled via LoggerBase